spec/components/inverse_header_spec.rb

```diff
@@ -36,4 +36,10 @@ def component_name
 
     assert_select ".gem-c-inverse-header--full-width"
   end
+
+  it "renders correct css class when padding-top flag is set to false" do
+    render(component_path, no_padding_top: true) { block }
```

Comment:
Again, this feels slightly more confusing than my sleep-deprived mind can handle. The test name says the flag is set to false, but the value given is true?